pkg/registry/core/node/strategy.go

```diff
@@ -68,9 +68,7 @@ func (nodeStrategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
 	node := obj.(*api.Node)
 	// Nodes allow *all* fields, including status, to be set on create.
 
-	if !utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) {
```

Comment:
why was the feature gate check removed?

Reply:
Restored, even though I am not sure why to keep it.